.github/actions/build-website/action.yml

```diff
@@ -60,7 +60,7 @@ runs:
         path: |
           .build-harness
           .cache
-        key: ${{ github.workflow }}
+        key: "build-website-cache" # Set same key to restore cache in all jobs
```

Comment:
I think there should be a label that if set, it will skip the cache.

E.g.

- rebuild
- bypass-cache
- no-cache

Reply:
added `no-cache` option